roles/matrix-bridge-appservice-slack/templates/systemd/matrix-appservice-slack.service.j2

```diff
@@ -0,0 +1,38 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Matrix Appservice Slack server
+{% for service in matrix_appservice_slack_systemd_required_services_list %}
+Requires={{ service }}
+After={{ service }}
+{% endfor %}
+{% for service in matrix_appservice_slack_systemd_wanted_services_list %}
+Wants={{ service }}
+{% endfor %}
+
+[Service]
+Type=simple
+ExecStartPre=-/usr/bin/docker kill matrix-appservice-slack
+ExecStartPre=-/usr/bin/docker rm matrix-appservice-slack
+
+# Intentional delay, so that the homeserver (we likely depend on) can manage to start.
+ExecStartPre=/bin/sleep 5
+
+ExecStart=/usr/bin/docker run --rm --name matrix-appservice-slack \
```

Comment:
Would be nice to make this `docker run` command more secure:
- using `--user={{ matrix_user_uid }}:{{ matrix_user_gid }}` or something similar -- running as a regular user, not `root`
- using `--cap-drop=ALL` to drop capabilities and additionally increase security